apps/vanilla/src/pages/exercise1/style/components.css

```
@@ -112,6 +123,21 @@
   border-color: var(--on-surface-color);
 }
 
+.search-input input::placeholder { /* Chrome, Firefox, Opera, Safari 10.1+ */
+  color: var(--on-surface-dim-color);
+  opacity: 1; /* Firefox */
+}
+
+/* stylelint-disable-next-line selector-pseudo-class-no-unknown */
+.search-input input:input-placeholder { /* Internet Explorer 10-11 */
+  color: var(--on-surface-dim-color);
+}
+
+/* stylelint-disable-next-line selector-pseudo-element-no-unknown */
+.search-input input::input-placeholder { /* Microsoft Edge */
+  color: var(--on-surface-dim-color);
+}
```

If you disable any rules, always leave comments why you are doing this.

```suggestion
/* It need for solve problem with placeholder in Microsoft Edge, Internet Explorer 10-11 */
/* stylelint-disable-next-line selector-pseudo-element-no-unknown */
.search-input input::input-placeholder,
.search-input input:input-placeholder {
  color: var(--on-surface-dim-color);
}
```